CMakeLists.txt

```diff
@@ -261,9 +261,9 @@ endif(NATS_BUILD_WITH_TLS)
 set(NATS_VERSION_MAJOR  3)
 set(NATS_VERSION_MINOR  10)
 set(NATS_VERSION_PATCH  0)
-set(NATS_VERSION_SUFFIX "-beta")
+set(NATS_VERSION_SUFFIX "")
 
-set(NATS_VERSION_REQUIRED_NUMBER 0x030900)
+set(NATS_VERSION_REQUIRED_NUMBER 0x030A00)
```

Comment:
I don't recall that my original intent was to use hex numbers for major, minor, update, meaning it should probably be `0x031000`. What matters is that the library is able to detect that the number from the header is greater/smaller or not than the one from the library.

Reply:
@kozlovic I think it's doing the right thing now (hex), see https://github.com/nats-io/nats.c/blob/f356e634faf2bf4c37ce7d092dc056ff17e55784/src/nats.c#L170 and https://github.com/nats-io/nats.c/blob/f356e634faf2bf4c37ce7d092dc056ff17e55784/test/test.c#L16604, or am I misinterpreting?

Reply:
@kozlovic also, if I set it to `0310` the test fails